### ISSUE TITLE
[FR] Precompiled Ender 3 firmware with PID Bed tuning enabled

### ISSUE THREAD

**Issue by bluriest** [bigtreetech/BIGTREETECH-SKR-mini-E3#537]
Please add PID bed tuning to the precompiled firmwares, it can make a huge difference in the quality of the prints

**Comment by RetArsonFD** [comment]
YES, YES, PLEASE !!!!!!!  I've been trying to compile firmware with this enabled for 2 weeks now.  All I keep getting are various error codes.

**Comment by decryptmars** [comment]
It really doesn't make any sense to me why PID bed tuning is DISABLED by default in the pre-compiled SKR mini e3 v2 firmwares. Why?
